crates/net/src/p2p_network.rs

```diff
@@ -59,56 +60,60 @@ impl P2pNetwork {
-                    BackendConfig::Lib3h(config) => config.clone(),
-                    _ => return Err(format_err!("mismatch backend type, expecting lib3h")),
-                };
+                unimplemented!()
+                // let backend_config = match &p2p_config.clone().backend_config {
+                //     BackendConfig::Lib3h(config) => config.clone(),
+                //     _ => return Err(format_err!("mismatch backend type, expecting lib3h")),
+                // };
 
-                Box::new(move |h| {
-                    Ok(
-                        Box::new(Lib3hWorker::with_wss_transport(h, backend_config.clone())?)
-                            as Box<dyn NetWorker>,
-                    )
-                })
+                // Box::new(move |h| {
+                //     Ok(
+                //         Box::new(Lib3hWorker::with_wss_transport(h, backend_config.clone())?)
+                //             as Box<dyn NetWorker>,
+                //     )
+                // })
             }
             // Create an InMemoryWorker Ghost Engine Worker
-            P2pBackendKind::GhostEngineMemory => Box::new(move |h| {
-                let backend_config = match &p2p_config.clone().backend_config {
-                    BackendConfig::Memory(config) => config.clone(),
-                    _ => return Err(format_err!("mismatch backend type, expecting memory")),
-                };
-                Ok(Box::new(Lib3hWorker::with_memory_transport(
-                    h,
-                    backend_config.clone(),
-                )?) as Box<dyn NetWorker>)
+            P2pBackendKind::GhostEngineMemory => Box::new(move |_h| {
+                unimplemented!()
+                // let backend_config = match &p2p_config.clone().backend_config {
+                //     BackendConfig::Memory(config) => config.clone(),
+                //     _ => return Err(format_err!("mismatch backend type, expecting memory")),
+                // };
+                // Ok(Box::new(Lib3hWorker::with_memory_transport(
+                //     h,
+                //     backend_config.clone(),
+                // )?) as Box<dyn NetWorker>)
             }),
 
             // Create an InMemoryWorker
             P2pBackendKind::LegacyInMemory => Box::new(move |h| {
                 Ok(Box::new(InMemoryWorker::new(h, &backend_config_str)?) as Box<dyn NetWorker>)
             }),
             // Create an Sim1hWorker
-            P2pBackendKind::SIM1H => Box::new(move |h| {
-                let backend_config = match &p2p_config.clone().backend_config {
-                    BackendConfig::Sim1h(config) => config.clone(),
-                    _ => return Err(format_err!("mismatch backend type, expecting sim1h")),
-                };
-                Ok(Box::new(Sim1hWorker::new(h, backend_config)?) as Box<dyn NetWorker>)
+            P2pBackendKind::SIM1H => Box::new(move |_h| {
+                unimplemented!()
```

Comment:
I know we removed the sim1h test suite. Are we (did we?) officially and fully deprecate sim1h?

Reply:
Let's discuss that today, and what that should look like

Reply:
Oops, forgot to bring it up today. Tomorrow!